adam/perception/high_level_semantics_situation_to_developmental_primitive_perception.py

```diff
@@ -730,25 +773,27 @@ def _perceive_ground_relations(self):
                             ):
                                 object_is_on_something = True
                     if not object_is_on_something:
```

Comment:
I wanted to chime in here with the reminder that we want to handle objects going _in_ something as well.
<img width="1044" alt="Screen Shot 2020-02-27 at 4 01 29 PM" src="https://user-images.githubusercontent.com/15387865/75486463-8de99880-597a-11ea-9501-636e5386f730.png">

Reply:
Good catch, thanks @elizlee 

I think it's safe to assume if something is `in` an object the default assumption should be the ground isn't supporting it. (Counterexample: A ball in an upsidedown cup) but that feels like a situation where we should template the `on(ball, ground)` relationship rather than assuming the default is handled that way.

Reply:
Fixed:
![image](https://user-images.githubusercontent.com/44595288/75489085-842b5980-59a9-11ea-836e-f105d125dc5f.png)